src/panels/lovelace/header-footer/hui-graph-header-footer.ts

```diff
@@ -100,10 +112,21 @@ export class HuiGraphHeaderFooter extends LitElement
 
   static get styles(): CSSResult {
     return css`
+      paper-spinner {
+        position: absolute;
+        top: calc(50% - 28px);
+        left: calc(50% - 12px);
+      }
+      .container {
```

Comment:
```suggestion
      .container {
        display: flex;
        justify-content: center;
```

Reply:
You can remove the `left`, `text-align` and `width` from the `.info` and `paper-spinner`